### DISCUSSION TITLE
Passing classes to VList

### DISCUSSION THREAD

**Discussion by bingud** [inokawa/virtua#537] [Q&A] [answered]
Hello, I can pass styles to a VList with the `style` prop, is there also a way to pass classes?

**Comment by inokawa** [comment, marked as answer]
You can pass `class`(or `className` in react) prop to VList too. If not, probably it's a bug.

**Reply by bingud** [reply]
thanks, i missed that. i read the VList code and didnt understand the `...rest` that is placed in the VList
